setup

```diff
@@ -53,7 +53,7 @@ function print_status_nl() {
 
 # Make sure only non-root users can run our script
 function quit_if_script_was_run_as_root() {
-  if [[ $EUID -eq 0 ]]; then
+  if  [[ ! -f /.dockerenv && $EUID -eq 0 ]]; then
```

Comment:
added this to the if statement because in the container `setup` is ran with `sudo` by default